charts/spring-service/templates/pre-deployment/secret.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: "kubernetes-client.io/v1"
+kind: ExternalSecret
+metadata:
+  namespace: {{.Values.namespace }}
+  name: {{ .Values.serviceName }}-external-secret
+spec:
+  backendType: systemManager
+  roleArn: {{ required "secretsRoleArn must be set!" .Values.secretsRoleArn }}
+  data:
+    - name: "sentry/dsn-{{ .Values.serviceName }}"
+      key: "/{{ required "clusterName must be set!" .Values.clusterName }}/sentry/dsn-{{ .Values.serviceName }}"
+    - name: "authenticorn/public-key"
+      key: "/{{ .Values.clusterName }}/authenticorn/public-key"
+  {{- if .Values.env.springDatasourceFromSecret }}
+    - name: "{{ .Values.env.springDatasourceFromSecret }}/jdbc-url"
+      key: "/{{ .Values.clusterName }}/{{ .Values.env.springDatasourceFromSecret }}/jdbc-url"
```

Comment:
ich hab übrigens hier überall ein vier-Gespann migriert "hostname" gab es auch immer. Eventuell kann das fast überall aus dem SSM rausfallen, wenn es hier nicht verwendet wird.

Reply:
Hostname sollte drin bleiben, das nutzen wir für die Zugangsdaten über `psql-rds-auth`

Reply:
Zur Not kann man es aber auch dann einfach in PSQL-RDS Auth aus der URL rausparsen.

Reply:
nö, kann man wegen mir schon behalten, ich hatte mich nur gefragt, wer das überhaupt benutzt.